draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +711,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+HPKE makes the following assumptions about its cryptographic
+dependencies.
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a Random Oracle.
+- Expand: Behaves as a Pseudorandom Function wherein the first
+argument is the key.
+
+For the DH-based KEM, HPKE also assumes the gap Computational
+Diffie-Hellman (CDH) problem is hard {{S01}}.
+
+HPKE has several security goals, depending on the mode of operation,
+against active and adaptive attackers that can compromise partial secrets
+of senders and recipients. The desired security goals are detailed below:
+
+- Message secrecy: Privacy of the sender's messages, i.e., IND-CCA2
+security.
+- Export key secrecy: Indistinguishability of the export secret from
+a uniformly random bitstring of equal length.
+- Sender authentication: Proof of sender origin for Auth and AuthPSK modes.
+
+The analysis considers two variants of HPKE usage: single-shot message
+encryption and secret key export. In the single-shot variant, S uses the
+single-shot API to use the key once to encrypt a plaintext. The export variant
+is the same as single-shot variant, except that the sender additionally exports
+two independent secrets using the secret export interface. We distinguish
+these two variants as the single-shot API does not lend itself to use
+the Export interface.s
+
+The table below summarizes the main results from {{HPKEAnalysis}}. N/A
+means that a property does not apply for the given mode, whereas X means
+the given mode satisfies the property.
+
+| Variant | Message Secrecy  | Export Key Secrecy | Sender Auth. |
+|:-------|:------------|:-----|:-------------|
+| Base, single-shot | X | N/A | N/A |
+| Base, export | X | X | N/A |
+| PSK, single-shot | X | N/A | X |
+| PSK, export | X | X | X |
+| Auth, single-shot | X | N/A | X |
+| Auth, export | X | X | X |
+| AuthPSK, single-shot | X | N/A | X |
+| AuthPSK, export | X | X | X |
+
+In this section, we consider a security issue that may arise in practice
+and an advanced use case.
```

Comment:
Is this sentence still relevant?  Doesn't seem like it.  If we are still discussing an advanced case, put a subsection break before it.

Reply:
I deleted it and wrapped this section in its own header "Security Properties"